packages/firebase_core/firebase_core_web/lib/src/interop/utils/utils.dart

```diff
@@ -113,7 +113,7 @@ PromiseJsImpl<S> handleFutureWithMapper<T, S>(
 ) {
   return PromiseJsImpl<S>(allowInterop((
     void Function(S) resolve,
-    void Function(Object) reject,
+    Null Function(Object) reject,
```

Comment:
```suggestion
    FutureOr<void> Function(Object) reject,
```

Shouldn't this be `FutureOr<void>`? Note `void` used instead of `Null` for [`prefer_void_to_null`](https://dart-lang.github.io/linter/lints/prefer_void_to_null.html)

Reply:
I thought about it a bit more, updated it to use a lambda instead so the type annotation can stay the same :)